src/password/scripts/directives.js

```diff
@@ -90,11 +90,12 @@
                         .then(function (response) {
                             scope.fields = response.data.actions.PUT;
                         }, function (response) {
-                            // If the response returns a 404 and we have a token.
-                            if (response.status === 404 && angular.isDefined(TOKEN)) {
-                                // There was a problem with the token.
+                            if (angular.isDefined(TOKEN)) {
                                 scope.tokenError = true;
                             }
+                            if (angular.isDefined(response.data) && angular.isDefined(response.data.detail)) {
+                                scope.errorMessage = response.data.detail;
```

Comment:
This is what `restframework` says about [errors](http://www.django-rest-framework.org/api-guide/exceptions/) such as `50x` and `40x` (except `400`):

> REST framework will return a response with an appropriate status code and content-type. The body of the response will include any additional details regarding the nature of the error.

Maybe this piece of code can be generic and apply to every request (or at least to every request happening in `angular-user-management`?